adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/impl/LoraProtocolAdapter.java

```diff
@@ -180,7 +182,7 @@ protected void customizeDownstreamMessage(final Message downstreamMessage, final
 
     void handleProviderRoute(final HttpContext ctx, final LoraProvider provider) {
 
-        LOG.debug("processing request from provider [name: {}, URI: {}", provider.getProviderName(), provider.pathPrefix());
+        LOG.debug("processing request from provider [name: {}", provider.getProviderName());
```

Comment:
we can still include the URI that has been invoked. We only need to retrieve it from the HttpContext instead of the LoraProvider ...